HandlingResources.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+class HandlingResources
+{
+  private $xml_data = null;
+  public $init_arr = [];
+  public $data = [];
+
+  private function replaceKey($arr, $old_key, $new_key)
+  {
+    $keys = array_keys((array) $arr);
+    if (array_key_exists($old_key, (array) $arr)) {
+      $keys[array_search($old_key, $keys)] = $new_key;
+    }
+    return $arr = array_combine($keys, (array) $arr);
+  }
+
+  public function formatting($init_arr, $arr)
+  {
+    foreach ($init_arr as $nested) {
+
+      $nested = (array) $nested;
+      if (strpos($nested['Rate'], '*') === 0) {
+        $nested['Rate'] = strlen($nested['Rate']);
+      }
+      $nested = $this->replaceKey($nested, 'Fare', 'Price');
+      $nested = $this->replaceKey($nested, 'Hotel', 'hotelName');
+      $nested = $this->replaceKey($nested, 'amenities', 'roomAmenities');
+      $arr[] = $nested;
+    }
+    return $arr;
+  }
+
+  public function sorting($data)
+  {
+    $count = count($data);
+    for ($i = 0; $i < $count; $i++) {
+      for ($j = $i + 1; $j < $count; $j++) {
+        if ($data[$i]['Rate'] < $data[$j]['Rate']) {
+          $temp = $data[$i];
+          $data[$i] = $data[$j];
+          $data[$j] = $temp;
+        }
+      }
+    }
+    return $data;
+  }
+
+  private function arrayToXml($data, $xml_data)
+  {
+    foreach ($data as $key => $value) {
+      if (is_array($value)) {
+        if (is_numeric($key)) {
+          $key = 'item' . $key;
+        }
+        $sub_node = $xml_data->addChild($key);
+        $this->arrayToXml($value, $sub_node);
+      } else {
+        $xml_data->addChild("$key", htmlspecialchars("$value"));
+      }
+    }
+  }
+
+  public function formatOutput()

```

Comment:
This method violates O in the SOLID principle as if you wanted to add "Yaml" format for example you will need to add a new if condition.
Can you please read about the strategy pattern?
https://refactoring.guru/design-patterns/strategy
Try to apply it to solve this problem